backend/framework/qlf/dashboard/bokeh/plots/plot2d/main.py

```diff
@@ -116,18 +116,42 @@ def circle(
             hover_color=hover_color,
             hover_alpha=hover_alpha,
             hover_line_color=hover_line_color,
-            radius=radius,
+            #radius=radius,
```

Comment:
If this is not going to be used please remove all occurrences of radios on `.circle`

Reply:
Done.